config/runtime/http.go

```diff
@@ -22,10 +22,11 @@ type ServerMux struct {
 
 // HTTPConfig represents the configuration of the ingress HTTP server.
 type HTTPConfig struct {
-	HealthPath string `env:"health_path"`
-	ListenPort int    `env:"default_port"`
-	UseXFH     bool   `env:"xfh"`
-	Timings    HTTPTimings
+	HealthPath      string `env:"health_path"`
+	ListenPort      int    `env:"default_port"`
+	UseXFH          bool   `env:"xfh"`
+  RequestIDFormat string `env:"request_id_format"`
```

Comment:
This looks like a missing `go fmt` (merge conflicts?).

Reply:
Yes, this must be from resolving the merge conflicts :smile: